src/wrappers/msvc_wrapper.cpp

```diff
@@ -103,7 +109,21 @@ string_list_t make_preprocessor_cmd(const string_list_t& args) {
 }
 }  // namespace
 
-msvc_wrapper_t::msvc_wrapper_t(const string_list_t& args) : program_wrapper_t(args) {
+msvc_wrapper_t::msvc_wrapper_t(const string_list_t& args) : program_wrapper_t(m_real_args) {
+  // Copy args to m_real_args, parsing any possible response file.
+  for (const auto& arg : args) {
+    if (arg.substr(0, 1) == "@") {
+      std::ifstream response_file(arg.substr(1));
+      if (response_file.is_open()) {
+        std::string line;
+        while (std::getline(response_file, line)) {
+          m_real_args += string_list_t::split_args(line);
```

Comment:
Note that this will not support recursive resource files, but MSVC doesn't support those either.  https://docs.microsoft.com/en-us/cpp/build/reference/at-specify-a-compiler-response-file?view=vs-2019